init_scripts/init-root.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+echo "nameserver 8.8.8.8" | sudo tee /etc/resolv.conf
+# Stay up-to-date
+apt-get -y update
+apt-get -y install python3-pip
```

Comment:
you should try to use a fixed version so that you're guaranteed to have the same environment every time you start it.